### ISSUE TITLE
About ClickMe dataset used in the paper

### ISSUE THREAD

**Issue by study0098** [serre-lab/Adversarial-Alignment#2]
Hi!

Thanks for this great work, it's really interesting. I have a question about the ClickMe dataset used in the paper: As said in the paper, the dataset contains nearly 200000 images. However, when I download the data from https://connectomics.clps.brown.edu/tf_records/clicktionary_files/, I find that the training data contains 325644 images. Could you please let me know the difference between the data used in your paper and the data downloaded as aforementioned? Moreover, are all the training samples of ClickMe datasets from the ImageNet training set, and is there any mapping between them? Thanks in advance for your help! I do apologize if these questions are repetitive.

**Comment by drewlinsley** [comment]
Thanks for the message! Our original ClickMe dataset contains clickmaps for a subset of all images in ImageNet. To harmonize models with these data, we only compute the alignment loss on those images where we have clickmaps. We have TFRecords with the full ImageNet/ClickMe dataset compiled together. I will dig it up — stay tuned.

**Comment by drewlinsley** [comment]
All ImageNet images + ClickMe maps (on a subset of all of them)
https://connectomics.clps.brown.edu/tf_records/archive/clickme_train.tfrecords
https://connectomics.clps.brown.edu/tf_records/archive/clickme_train_meta.npz

https://connectomics.clps.brown.edu/tf_records/archive/clickme_val.tfrecords
https://connectomics.clps.brown.edu/tf_records/archive/clickme_val_meta.tfrecords

https://connectomics.clps.brown.edu/tf_records/archive/clickme_test.tfrecords
https://connectomics.clps.brown.edu/tf_records/archive/clickme_test_meta.tfrecords
